_posts/2021-12-31-happy-new-year-2022-recap.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Happy New Year and a Recap of 2021"
+summary: ""
+tags: [news, community]
+author: "Reuben Yap"
+img: "/blog/assets/firo-happy-2022.png"
+---
+The Firo Core team would like to wish everyone a Happy New Year and take this opportunity to thank everyone for their support and belief in our work!
+
+Now is also a good time to have a quick recap of the many exciting things we achieved this year.
+
+## Tech
+
+[Lelantus launched and activated](https://firo.org/2021/01/15/lelantus-activated-mainnet.html)
+[Chainlocks activated](https://firo.org/2021/01/28/chainlocks-activated-mainnet.html)
+[Binance Smart Chain integration](https://firo.org/2021/04/01/firo-bsc-integration.html)
+[RAP addresses launched](https://firo.org/2021/06/09/introducing-receiver-address-privacy-for-firo.html)
+[Lelantus Spark, our next gen privacy protocol revealed](https://firo.org/2021/08/24/presenting-lelantus-spark.html)
+[FiroPoW launched](https://firo.org/2021/10/01/firopow-and-instantsend-release.html)
+[Instant Send activated](https://firo.org/2021/10/01/firopow-and-instantsend-release.html)
+[Firo Mobile Wallet with Lelantus by default](https://firo.org/2021/12/16/firo-mobile-wallet-release-with-lelantus.html)
+[MTP Data Stripping](https://firo.org/2021/12/19/mtp-stripping-live-on-firo.html)
+[New wallet GUIs for Electron and QT](https://firo.org/2021/08/27/firo-client-release-210.html)
+[Elysium Privacy Infrastructure Layer Testnet](https://firo.org/2021/12/22/elysium-testnet-and-gui.html)
+[FiroDex Beta launch](https://githubplus.com/firoorg/FiroDEX-Desktop)

```

Comment:
Change to Github.com link

Reply:
Changed to https://github.com/firoorg/FiroDEX-Desktop